openapi/parameters.yaml

```diff
@@ -225,3 +225,24 @@ user_id:
   in: path
   schema:
     type: string
+JournalBibstem:
+  name: bibstem
+  description: ADS identifier for publications embedded within bibcodes
```

Comment:
The wording is a little confusing here - maybe it just needs some punctuation. What about: `ADS identifier for publications (embedded within bibcodes)` or `ADS identifier for publications, embedded within bibcodes`

Reply:
Fixed.